packages/plugin-phone/src/call.js

```diff
@@ -530,11 +542,19 @@ const Call = SparkPlugin.extend({
       }
     }
 
-    this.spark.phone.register()
+    this.logger.info(this.registered, !!this.spark.device.url, this.connected, this.spark.mercury.connected, this.spark.device.url);
+
+    // Note: mercury.connect() will call device.register() if need. We're not
+    // using phone.register() here because it guarantees a device refresh, which
+    // is probably unnecessary.
+    this.spark.mercury.connect()
```

Comment:
if phone.register is called, can it handle not having a Client_secret and fail gracefully?